dockers/docker-platform-monitor/docker-pmon.supervisord.conf.j2

```diff
@@ -79,7 +79,11 @@ dependent_startup_wait_for=start:exited
 
 {% if not skip_xcvrd %}
 [program:xcvrd]
+{% if delay_xcvrd %}
```

Comment:
what if delay_xrcvd is not defined? will this throw exception?

Reply:
Just did a quick check. It won't throw exception if delay_xcvrd is not defined.